app/assets/javascripts/channels/event_channel.coffee

```diff
@@ -0,0 +1,20 @@
+class App.EventChannel
```

Comment:
Intended to abstract the event channel into it's own class so we can use it where needed more readily